src/routes/UserRoutes.ts

```diff
@@ -1,27 +1,56 @@
 import { Response } from "express";
 import { UserService } from "@src/services";
 import { HttpStatusCodes } from "@src/common";
-import { AddUserRequest, LoginUserRequest } from "@src/types";
+import {
+  AddUserRequest,
+  LoginUserRequest,
+  EmailVerificationRequest,
+} from "@src/types";
 import { pinoLogger } from "@src/logger";
 
 /**
  * Add one user.
  */
 async function add(req: AddUserRequest, res: Response): Promise<void> {
   const { body } = req;
-  const [user, error] = await UserService.add(body);
+  const [data, error] = await UserService.add(body);
 
   if (error) {
-    pinoLogger.warn({ error: error.message }, "Error in add user route");
+    pinoLogger.warn({ error: error.message }, "Error in add UserRoutes");
 
     res
       .status(error.httpStatusCode)
       .send({ errors: error.buildErrorPayload() });
     return;
   }
 
-  pinoLogger.info("Created user is sending to the client");
-  res.status(HttpStatusCodes.CREATED).send(user);
+  pinoLogger.info("Data is sending to the client add UserRoutes");
+  res.status(HttpStatusCodes.CREATED).send(data);
+}
+
+/**
+ * User email verification.
+ */
+async function emailVerification(
+  req: EmailVerificationRequest,
+  res: Response
+): Promise<void> {
+  const { hash } = req.query;
+  const [, error] = await UserService.emailVerification(hash);
+
+  if (error) {
+    pinoLogger.warn(
+      { error: error.message },
+      "Error in emailVerification route"
+    );
+    res
+      .status(error.httpStatusCode)
+      .send({ errors: error.buildErrorPayload() });
+    return;
+  }
+
+  pinoLogger.info("Email successfully verified");
+  res.status(HttpStatusCodes.OK).send("OK");
```

Comment:
I would maybe return 

```suggestion
  res.status(HttpStatusCodes.OK).send({ message: "Email successfully verified.", success: true });
```